templates/var/backups/openldap/config.ldif

```diff
@@ -100,6 +107,14 @@ objectClass: olcMdbConfig
 olcDatabase: {1}mdb
 olcDbDirectory: /usr/local/openldap/var/openldap-data
 olcSuffix: {{ ldaptoolbox_openldap_suffix }}
+{% if ldaptoolbox_openldap_database_olcAccess|length > 0 %}
```

Comment:
This was not needed because all the ACLs where defined in frontend backend.

This is ok because it adds a new way to do, but may be very confusing for admins understanding which rule is applied. (and which has precedence)